x/interchainstaking/keeper/delegation.go

```diff
@@ -168,6 +168,15 @@ func (k *Keeper) RemoveDelegation(ctx sdk.Context, chainID string, delegation ty
 	return nil
 }
 
+// RemovePerformanceDelegation removes a performance delegation.
+func (k *Keeper) RemovePerformanceDelegation(ctx sdk.Context, chainID string, delegation types.Delegation) error {
```

Comment:
Nice catch 💪 @DongLieu. Can you set up testing for the new function you added, maybe like [this](https://github.com/quicksilver-zone/quicksilver/pull/758/RemovePerformanceDelegation)

Reply:
it's available [here:](https://github.com/quicksilver-zone/quicksilver/pull/678#issuecomment-1786296197) pull #678

Reply:
Ok, I will add it now ,separately.